ROX-17465: Add RHACS Workload CVE info (#61387)
openshift/openshift-docs: modules/vulnerability-management20-view-cve.adoc

// Module included in the following assemblies:
//
// * operating/manage-vulnerabilities.adoc
:_content-type: PROCEDURE
[id="vulnerability-management20-view-workload-cve_{context}"]
= Viewing workload CVEs in Vulnerability Management (2.0)

[role="_abstract"]
You can view a comprehensive list of vulnerabilities, or CVEs, in {product-title-short} across images and deployments. You can use the search filter bar to select specific CVEs, images, deployments, namespaces, or clusters.

:FeatureName: Vulnerability Management 2.0
include::snippets/technology-preview.adoc[]

.Procedure
. In the {product-title-short} portal, navigate to *Vulnerability Management 2.0* -> *Workload CVEs*.

> . From the drop-down list, select the search criteria you want to use. You can select an item type, such as a cluster, from the drop-down list, and then select the specific name of the item. You can add additional items to the filter by selecting another item from the drop-down list and selecting the specific name of the new item. For example, you can select a specific image and a specific cluster to limit results to those selections. You can filter on the following items:

Author: No label. I'll add a pic below. You choose either Deployment, CVE, Image, Namespace, or cluster from the list.

> . From the drop-down list, select the search criteria you want to use. You can select an item type, such as a cluster, from the drop-down list, and then select the specific name of the item. You can add additional items to the filter by selecting another item from the drop-down list and selecting the specific name of the new item. For example, you can select a specific image and a specific cluster to limit results to those selections. You can filter on the following items:

Reviewer: In that case, I think I'd keep "drop-down list" for the first mention, but just "list" in subsequent mentions, assuming that within the context, the reader will know which list you're referring to.

* CVE
* Image
* Deployment
* Namespace
* Cluster
. Optional: Use the *CVE severity* drop-down list to select the severities of the CVEs that you want to display.

> . Click the desired button to view a list of vulnerabilities, images, or deployments in the system.

Author: Relevant is great, I was drawing a blank on this!

+
[NOTE]
====
The *Filtered view* icon indicates that the results displayed have been filtered based on the criteria that you selected. You can click *Clear filters* to remove all filters, or remove individual ones by clicking on them.
====

> . Click on a CVE, image name, or deployment name in the list of results to view more information about the item. For example, depending on the item type, you can view information such as if a CVE is fixable, if an image is active, the Dockerfile line in the image that contains the CVE, and external links to information about the CVE in Red Hat and other CVE databases.

Reviewer: ```suggestion
. In the list of results, click a CVE, image name, or deployment name to view more information about the item. For example, depending on the item type, you can view the following information:
+
* Whether a CVE is fixable
* Whether an image is active
* The Dockerfile line in the image that contains the CVE
* External links to information about the CVE in Red Hat and other CVE databases
```
